src/auth-service/utils/create-user.js

```diff
@@ -187,7 +187,6 @@ const join = {
               success: true,
               message: "Successfully fetched user data",
               status: httpStatus.OK,
-              data: userRecord,
```

Comment:
thanks @BenjaminSsempala for this.

In the corresponding `controller` logic, we might want to handle this removal accordingly to prevent runtime errors.

**The 3 Alternatives are listed below:**

- ALT 1: Within this create-user **util**, just assign data to an empty array. More like `data:[]` and stop at that.
- ALT 2: Within the create-user **controller** on line 466, just remove the user field as shown below:

 
```
  if (responseFromCreateUser.success === true) {
        const status = responseFromCreateUser.status
          ? responseFromCreateUser.status
          : httpStatus.OK;
        return res.status(status).json({
          success: true,
          message: responseFromCreateUser.message,
        });
      }
```

Reply:
- ALT 3: Still on line 466 of the create-user **controller**, alternatively just do some conditional statement as shown below:

```
    if (responseFromCreateUser.success === true) {
        const status = responseFromCreateUser.status
          ? responseFromCreateUser.status
          : httpStatus.OK;
        return res.status(status).json({
          success: true,
          message: responseFromCreateUser.message,
          user: responseFromCreateUser.data? responseFromCreateUser.data: [],
        });
      }
```

Reply:
Thank you @Baalmart .
I think I'll use the first alternative.

Reply:
@Baalmart  Meanwhile alternative 2 and 3 seem to be targerting `responseFromcreateUser` 
Shouldn't the change be targerting the `lookUpFirebaseUser` function, i.e  to remove the userfield from the create-user controller at line 199.
`await createUserUtil.lookUpFirebaseUser(request, (result) => {
        if (result.success === true) {
          const status = result.status ? result.status : httpStatus.OK;
          res.status(status).json({
            success: true,
            message: result.message,
            exists: true,
            status: "exists",
          });`

Reply:
That sounds right @BenjaminSsempala , thanks! That sounds about right.